src/cpu/x64/jit_uni_pooling.cpp

```diff
@@ -1477,6 +1477,9 @@ template struct jit_uni_pooling_fwd_t<avx512_core, data_type::bf16>;
 template struct jit_uni_pooling_bwd_t<avx512_core, data_type::bf16>;
 template struct jit_uni_pooling_fwd_t<avx512_core_fp16, data_type::f16>;
 template struct jit_uni_pooling_bwd_t<avx512_core_fp16, data_type::f16>;
+template struct jit_uni_pooling_fwd_t<sse41, data_type::u8>;
+template struct jit_uni_pooling_fwd_t<avx2, data_type::u8>;
+template struct jit_uni_pooling_fwd_t<avx512_core, data_type::u8>;
```

Comment:
An extra task might be to get rid of data type template argument from the implementation which would save on same disk space.